### DISCUSSION TITLE
[WiX4] WixUI: Next button does nothing

### DISCUSSION THREAD

**Discussion by jan-tosovsky-cz** [wixtoolset/issues#6549] [Questions]
My goal is to show a dialog before the installation and another after the installation finishes.

To my config I've added this specification based on minimal configuration example here https://github.com/wixtoolset/UI.wixext/blob/master/src/wixlib/WixUI_Minimal.wxs (not sure it is applicable for V4)
```
        <UI Id="WixUI_Minimal">
            <Property Id="WixUI_Mode" Value="Minimal" />

            <DialogRef Id="ErrorDlg" />
            <DialogRef Id="FatalError" />
            <DialogRef Id="FilesInUse" />
            <DialogRef Id="MsiRMFilesInUse" />
            <DialogRef Id="PrepareDlg" />
            <DialogRef Id="ProgressDlg" />
            <DialogRef Id="ResumeDlg" />
            <DialogRef Id="UserExit" />
            <DialogRef Id="WelcomeDlg" />
            <DialogRef Id="WelcomeEulaDlg" />
        </UI>
        <UIRef Id="WixUI_Common" />
```
It is built properly when referencing the WixUI extension:
`wix build -ext C:\my-project\.wix\extensions\WixToolset.UI.wixext\4.0.0-preview.0\tools\WixToolset.UI.wixext.dll installer.wxs`

When installer is executed, the Welcome dialog is shown, but the Next button does nothing. However, Cancel button works as expected (asking whether really exit or not).

Could be related: There are two msiexec.exe instances in Task manager when installer is started. One running under my user account while another for SYSTEM account. When installation is cancelled, only that user instance is closed. That SYSTEM is still running (until killed manually).

Is this a configuration problem or rather something in the engine? Can I somehow enable logging to triage the root cause?

**Comment by robmen** [comment]
There may be several msixexec.exe processes running and they run for a while after an install is completed. This is normal Windows Installer behavior.
